chainio/clients/elcontracts/builder.go

```diff
@@ -0,0 +1,49 @@
+package elcontracts
+
+import (
+	"github.com/Layr-Labs/eigensdk-go/chainio/clients/eth"
+	"github.com/Layr-Labs/eigensdk-go/chainio/txmgr"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/Layr-Labs/eigensdk-go/metrics"
+)
+
+func BuildClients(
+	config Config,
+	ethHttpClient eth.Client,
+	txMgr txmgr.TxManager,
+	logger logging.Logger,
+	eigenMetrics *metrics.EigenMetrics,
+) (*ELChainReader, *ELChainWriter, *ContractBindings, error) {
+	elContractBindings, err := NewBindingsFromConfig(
+		config,
+		ethHttpClient,
+		logger,
+	)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	elChainReader := NewELChainReader(
```

Comment:
didn't we have new names for these without EL? `elcontracts.NewChainReader`?

Reply:
updated

Reply:
lol sorry for no commit info :P